lib/puppet/type/service.rb

```diff
@@ -81,11 +81,17 @@ def retrieve
       end
 
       # This only makes sense on systemd systems. Static services cannot be enabled
-      # or disabled manually.
+      # or disabled manually. Indirect services should not be enabled or disabled.
```

Comment:
maybe be more specific here as to why they should not be enabled.
i.e.: `Indirect services should not be enabled or disabled due to limitations in systemd, see https://github.com/systemd/systemd/issues/6681`